hopp/simulation/technologies/financial/custom_financial_model.py

```diff
@@ -60,8 +60,8 @@ class SystemCosts(FinancialData):
     om_batt_fixed_cost: float = field(default=0)
     om_batt_variable_cost: float = field(default=[0])
     om_batt_capacity_cost: float = field(default=0)
-    om_batt_replacement_cost: float = field(default=0)
-    om_replacement_cost_escal: float = field(default=0)
+    om_batt_replacement_cost: float = field(default=0) # Replacement cost 1 [$/kWh] - not used in custom fin model
+    om_replacement_cost_escal: float = field(default=0) # Production-based O&M escalation [%/year] - not used in custom fin model
```

Comment:
Given that these aren't used in the custom financial model, can we generally remove them? Or does another model use them in some way?